### PR TITLE
Add en version of the quick start notebook.

### PR DESCRIPTION
**Pull request by g-votte**
This PR adds an English version of the hands-on material.

View in Colab: https://colab.research.google.com/github/g-votte/optuna-hands-on/blob/en/en/01_Optuna_Quick_Start.ipynb
View in GitHub: https://github.com/g-votte/optuna-hands-on/blob/en/en/01_Optuna_Quick_Start.ipynb

### CODE REVIEW

File: en/01_Optuna_Quick_Start.ipynb [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+    "\n",
+    "def objective(trial):\n",
+    "    some_machine_learning_logic()  # Write your machine learning logic here.\n",
+    "    return evaluation_score  # Return the evaluation score of the trained model.\n",
+    "\n",
+    "study = optuna.create_study(direction='minimize')\n",
+    "study.optimize(objective, n_trials=N_TRIALS)  # Specify the number of trials. \n",
+    "```\n",
+    "\n",
+    "## Optimize Quasratic Function\n",
```
Comment (sile) [inline en/01_Optuna_Quick_Start.ipynb]: s/Quasratic/Quadratic/
```suggestion
    "## Optimize Quadratic Function\n",
```

File: en/01_Optuna_Quick_Start.ipynb [hunk 2]
```diff
@@ -0,0 +1,358 @@
...
+    "Before optimizing a machine learning model, let's see how Optuna solves a very simple task that minimizes the output of $f(x) = (x - 2)^2$.\n",
+    "Although the answer obiously is $f(x) = 0$ when $x = 2$, Optuna doesn't know how to solve that."
```
Comment (sile) [inline en/01_Optuna_Quick_Start.ipynb]: s/obiously is/is obviously/
```suggestion
    "Although the answer is obviously $f(x) = 0$ when $x = 2$, Optuna doesn't know how to solve that."
```

File: en/01_Optuna_Quick_Start.ipynb [hunk 3]
```diff
@@ -0,0 +1,358 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "colab": {},
+    "colab_type": "code",
+    "id": "_uruvcV6IraQ"
+   },
+   "source": [
+    "## Imperative Interface: Search Conditional Hyperparameters\n",
```
Comment (sile) [inline en/01_Optuna_Quick_Start.ipynb:251]: I don't know the reason but this cell is rendered as a code block on colab (see the following snapshot):
image

It is fine on GitHub:
image

Comment (sile) [reply en/01_Optuna_Quick_Start.ipynb:251]: In my environment, this problem seems resolved by applying the following diff:
```diff
diff --git a/ja/01_Optuna_Quick_Start.ipynb b/ja/01_Optuna_Quick_Start.ipynb
index 420cb37..92efeb7 100644
--- a/ja/01_Optuna_Quick_Start.ipynb
+++ b/ja/01_Optuna_Quick_Start.ipynb
@@ -414,8 +414,7 @@
   {
    "cell_type": "markdown",
    "metadata": {
-    "colab": {},
-    "colab_type": "code",
+    "colab_type": "text",
     "id": "_uruvcV6IraQ"
    },
    "source": [

```

File: en/01_Optuna_Quick_Start.ipynb [hunk 4]
```diff
@@ -0,0 +1,358 @@
...
+    "\n",
+    "study = optuna.create_study(direction='minimize')\n",
+    "study.optimize(objective, n_trials=100)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Executing the cell above, you should see 100 lines of execution log. Optuna calls `objective` 100 times changing the value of `x`, where the range of `x` is specified as $[-100, 100]$ in `trial.suggest_uniform('x', -100, 100)`. A `trial` is an object passed by Optuna, corresponds to a single call of `objective`, and provides interfaces to get next hyperparameter to be tried.\n",
```
Comment (toshihikoyanase) [inline en/01_Optuna_Quick_Start.ipynb]: I think `[-100, 100]` may be confusing. It usually means `-100 <= x <= 100` but `trial.suggest_uniform` does not include `high`. What do you think of a description like `-100 <= x < 100`?

File: en/01_Optuna_Quick_Start.ipynb [hunk 5]
```diff
@@ -0,0 +1,358 @@
...
+    "Although the answer obiously is $f(x) = 0$ when $x = 2$, Optuna doesn't know how to solve that."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import optuna  # Don't forget installing optuna with `!pip install optuna`.\n",
```
Comment (toshihikoyanase) [inline en/01_Optuna_Quick_Start.ipynb]: I think `forget to install` may be suitable for the situation if this sentence means that please install optuna before you execute this code.

File: en/01_Optuna_Quick_Start.ipynb [hunk 6]
```diff
@@ -0,0 +1,358 @@
...
+    "print('Best parameter: ' + str(study.best_params))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In summary, you need the following steps to set up the optimization.\n",
+    "\n",
+    "- Define the objective function that calculate the minimization/maximization target.\n",
```
Comment (toshihikoyanase) [inline en/01_Optuna_Quick_Start.ipynb]: `calculate` seems to be a typo for `calculates`.

File: en/01_Optuna_Quick_Start.ipynb [hunk 7]
```diff
@@ -0,0 +1,358 @@
...
+    "\n",
+    "# output: evaluation score\n",
+    "print('Mean squared error: ' + str(error))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Performance of Lasso regression is sensitive to the L1 constant, `alpha`, and it's tiresome for humans to manually search this hyperparameter. With Optuna, you can search the appropriate value of `alpha` as follows. Note that you just need to wrap the machine learning logic in the previous cell with `objective` and to set up `study` object."
```
Comment (toshihikoyanase) [inline en/01_Optuna_Quick_Start.ipynb]: I'm not sure but maybe `find` or `search for` is more suitable for the sentence `you can search the appropriate value` than just `search`. In my understanding, this sentence means `you can search (the hyperparameter space) for the appropriate value`.

**Review by toshihikoyanase — APPROVED** [review]
LGTM with a nit.

This is not a critical point but I think you might overlook a comment. In `Optimize Machine Learning Models` section, I still find the expression `search the appropriate value` as I mentioned in https://github.com/pfnet-research/optuna-hands-on/pull/2#discussion_r300906687. Please fix it if you think it's relevant.

**Pull request merged**